Otherwise the checkout on our bamboo-agents fail due to:
"cvs [init aborted]: cannot get working directory: No such file or directory"

This is caused by CVS using its bundled getcwd instead of the musl one.

--- old/lib/xgetcwd.c
+++ new/lib/xgetcwd.c
@@ -21,12 +21,12 @@
 #ifdef HAVE_CONFIG_H
 # include <config.h>
 #endif
+#include <unistd.h>
 
 #include "xgetcwd.h"
 
 #include <errno.h>
 
-#include "getcwd.h"
 #include "xalloc.h"
 
 /* Return the current directory, newly allocated.
